articles/systems/01-camera-system.md

```diff
@@ -38,7 +38,7 @@ For the most part, you should not need to alter the camera system configuration
 
 ### Related Articles
 
-* [XRTK Platform System](08-platform-system.md)
+* [XRTK Platform System](08-platform-framework.md)
```

Comment:
Still not sure about the name of this article as there isn't a platform _system_
```suggestion
* [XRTK Platform framework](08-platform-framework.md)
```

Reply:
All references to Platform System updated to Platform Framework